index.js

```diff
@@ -29,22 +28,22 @@ module.exports = function(src, options = {}) {
   const dependencies = [];
 
   if (typeof src === 'undefined') {
-    throw new Error('src not given');
```

Comment:
Would this be considered a breaking change?

Reply:
Not sure, it depends. It's still an error, its type is more explicit is all.

Reply:
We can revert it if you prefer, I just have my linter set up this way (xo).

Reply:
I think it's safer to revert this particular change. Someone trying to sniff for Error won't catch the TypeError unfortunately.

Reply:
Done, just squash and merge when ready :)